src/python_interpreter/mod.rs

```diff
@@ -47,7 +47,7 @@ fn windows_interpreter_no_build(
         }
     }
 
-    let python_arch = match platform.as_str() {
+    let python_arch = match platform.as_str().trim() {
```

Comment:
This could be simpler.

```suggestion
    let python_arch = match platform.trim() {
```